apis/feedback.php

```diff
@@ -71,11 +72,15 @@ function validateName($name)
 
 if (validateName($name_user) && validaterating($rating) && validatesuggestions($suggestions) && validatecomments($comment_abt_event))
 {
-    $query2 = mysqli_query($con, "SELECT * from user where email='$email_id'");
+    $query2 = mysqli_query($con, "SELECT * from user where email='".$email_id."'");
     $count1 = mysqli_num_rows($query2);
     if ($count1 != 0)
     {
         $q1 = mysqli_query($con, "UPDATE user SET rating='$rating',comments_abt_event='$comment_abt_event',suggestions='$suggestions' WHERE  email ='" . $email_id . "'");
+        $query_row = mysqli_fetch_array($query2,MYSQLI_ASSOC);
+        $_SESSION['name']=$query_row['user_name'];
+        $_SESSION['admin_id']=$query_row['admin_fk'];
```

Comment:
Why is it admin id? its just a user id if I am not wrong.

Reply:
Foreign key is same as the Id of admin in admin table. So, I named it admin id.

Reply:
Admin has nothing to do with the user certificate thing. The foreign key is there to know that the user is giving feedback for which event. Also, while uploading the participant list by the organiser, the admin id field will be filled by. There can be multiple participants under a single admin id

Reply:
So, the unique id for the user is the 'user id'. You can make use of that.